src/environment.js

```diff
@@ -208,11 +214,18 @@ var Environment = Obj.extend({
                     }
                 }
                 else {
-                    var tmpl = new Template(info.src, _this,
+                    var tmpl;
+                    if(ignoreMissing && fileMissing) {
```

Comment:
this can be replaced with `if (!info)` -- though probably clearer to swap with the `else` clause, so you have `if (info) { create a real template } else { create a no-op template }`